react/components/SearchQuery.js

```diff
@@ -144,6 +145,9 @@ const useCorrectSearchStateVariables = (
 }
 
 const useQueries = (variables, facetsArgs) => {
+  const { getSettings } = useRuntime() // Is it okay to call this here?
```

Comment:
Yes, if it's a hook, no problem.
```suggestion
  const { getSettings } = useRuntime()
```